testing/maas_billing_tests_independent/tests/conftest.py

```
@@ -0,0 +1,251 @@
+            if r.status_code in (404, 405):
+                # endpoint not present or method not allowed → try next ep
+                break
+    return None
+
+def mint_maas_key(http: requests.Session, base_url: str, oc_user_token: str, minutes=10) -> str:
+    tok = _try_mint_maas_key(http, base_url, oc_user_token, minutes=minutes)
+    if tok:
+        return tok
+    raise AssertionError("Could not mint a MaaS key with any common body/endpoint variant")
+
+def revoke_maas_key(http: requests.Session, base_url: str, oc_user_token: str, token: str | None = None):
```

Ruff (0.14.0)

Unused function argument: `token`

(ARG001)